scripts/create_issues_versionchange_canonical.sh

```diff
@@ -0,0 +1,146 @@
+#!/bin/bash
+
+print_usage() {
+    echo ">>>>> Usage: ./canonical_data_check_open_issue.sh -t path/to/track -s path/to/problem/specifications"
+}
+
+# Execution begins
+
+command -v jq >/dev/null 2>&1 || {
+    echo >&2 ">>>>> This script requires jq but it's not installed. Aborting."
+    exit 1
+}
+
+num_args=$#
+
+if [[ $num_args = 0 ]]; then
+    print_usage
+    exit 0
+fi
+
+path_to_track=
+path_to_problem_specifications=
+
+while getopts ":t:s:" option; do
+    case "$option" in
+        "t")
+            path_to_track="$OPTARG"
+            ;;
+        "s")
+            path_to_problem_specifications="$OPTARG"
+            ;;
+        *)
+            echo ">>>>> Unrecognized option. Aborting."
+            print_usage
+            exit 1
+            ;;
+    esac
+done
+
+if [[ -z "$path_to_track" ]]; then
+    echo ">>>>> Path to track missing."
+    print_usage
+    exit 1
+fi
+
+if [[ -z "$path_to_problem_specifications" ]]; then
+    echo ">>>>> Path to problem specifications missing."
+    print_usage
+    exit 1
+fi
+
+config_file_path="$path_to_track/config.json"
+
+if ! [[ -f "$config_file_path" ]]; then
+    echo ">>>>> Config file not found at ${config_file_path}."
+    exit 1
+fi
+
+track_exercise_slugs=$(jq '.exercises[] | select(has("deprecated") | not) | .slug' "$config_file_path" | tr -d "\"" | sort)
+update_needed_count=0
+
+
+# Create a file named ".exercism-version-update-issue-script-settings.sh"
+# with the following variables and put in in your home directory
+#
+# TOKEN="your_token"
+# OWNER="your_username"
+# REPO="repo_name"
```

Comment:
Printing an error with a summary of this information if these values aren't present might be useful? 🙂

Reply:
Great idea! I will add the error message.